azuredeploy.json

```diff
@@ -83,6 +83,13 @@
         "description": "The name of the database to provision on the PostgreSQL server."
       }
     },
+    "databaseServerPort": {
```

Comment:
Given that Azure Postgres always has to use port 5432, this shouldn't be a parameter: could lead to user confusion if they pass a different port but nothing changes in the deployment. Let's move it to a variable instead.

Reply:
agreed.